python/lsst/ts/salobj/topics/read_topic.py

```diff
@@ -88,12 +88,23 @@ class ReadTopic(BaseTopic):
     max_history : `int`
         Maximum number of historical items to read:
 
-        * 0 is strongly recommended for commands and the ackcmd reader
+        * 0 is required for commands and the ackcmd reader
         * 1 is recommended for events and telemetry
     queue_len : `int` (optional)
         The maximum number of items that can be read and not dealt with
         by a callback function or `next` before older data will be dropped.
 
+    Raises
+    ------
+    ValueError
+        If max_history < 0.
+    ValueError
+        If max_history > 0 and the topic is volatile (command or ackcmd).
+    ValueError
+        If queue_len <= 0.
+    ValueError
```

Comment:
I agree that it seems wasteful to repeat the exception class. My goal is to have each individual error conditions clearly delineated. (I want to avoid one long sentence listing all possible conditions that would raise ValueError, since one can easily miss a condition). I suppose I could use a bullet list. But this works.

I apologize for the outdated docs; I just uploaded new ones (after moving the nonfunctional attribute docs to the class doc strings).